tests/integration/construct_component/dotnet/Component.cs

```diff
@@ -16,6 +16,9 @@ class Component : Pulumi.ComponentResource
     [Output("childId")]
     public Output<string> ChildId { get; private set; } = null!;
 
+     [Output("secret")]
```

Comment:
Super nitpick (I'm sorry):

```suggestion
    [Output("secret")]
```